build-package.sh

```diff
@@ -1,6 +1,21 @@
 #!/bin/bash
 # shellcheck disable=SC1117
 
+# Fixing the number of calls and the list of compiled packages
+if [[ ! "$TERMUX_BUILD_PACKAGE_CALL_DEPTH" =~ ^[0-9]+$ ]]; then
+	export TERMUX_BUILD_PACKAGE_CALL_DEPTH=0
+	export TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST=" "
+else
+	export TERMUX_BUILD_PACKAGE_CALL_DEPTH=$((TERMUX_BUILD_PACKAGE_CALL_DEPTH+1))
```

Comment:
Remove this variable, directly read from file everytime.

Reply:
Updated variable value cannot go back to parents calls, hence we use a shared file for `build-package.sh` for all its depths.